audbackend/core/backend/minio.py

```diff
@@ -398,3 +405,13 @@ def _size(
         path = self.path(path)
         size = self._client.stat_object(self.repository, path).size
         return size
+
+
+def _metadata():
```

Comment:
Is there a reason to maintain `_metadata` completely separate from the `Minio` class? 
Intuitively one would think of `_metadata` as a (private) property. On the other hand, the class needs to implement the interface methods from the parent class, and a property might become confusing.

Reply:
I tend to implement such helper functions independent of the class. My main reason is that in the class we only have the methods that needed to be added, and it does not get cluttered. But maybe it has also some drawbacks?